packages/@ourworldindata/grapher/src/core/GrapherAnalytics.ts

```diff
@@ -71,6 +73,14 @@ export class GrapherAnalytics {
     private version: string // Ideally the Git hash commit
     private isDev: boolean
 
+    logGrapherView(slug: string, view?: Record<string, string>): void {
+        this.logToGA({
+            event: EventCategory.GrapherView,
+            grapherPath: `/grapher/${slug}`,
+            grapherView: view,
```

Comment:
I *think* this needs to be 

```js
grapherView: JSON.stringify(view),
```

so that GA doesn't log `grapherView` as `[Object object]`. Sorry, wasn't thinking about this properly when we discussed this param yesterday.

Reply:
changing it. thanks for double-checking!